src/notation/view/notationpaintview.cpp

```diff
@@ -714,6 +714,14 @@ void NotationPaintView::keyReleaseEvent(QKeyEvent* event)
     }
 }
 
+bool NotationPaintView::event(QEvent* ev)
+{
+    if (ev->type() == QEvent::Type::ShortcutOverride) {
+        shortcutOverride(static_cast<QKeyEvent*>(ev));
```

Comment:
I suppose I could have called it handleShortcutOverride or shortcutOverrideEvent but this isn't a standard virtual override.